aspnetcore/fundamentals/servers/httpsys.md

```diff
@@ -0,0 +1,146 @@
+
+The most common tasks you need to use netsh.exe for are reserving URL prefixes and assigning SSL certificates.
+
+NetSh.exe is not an easy tool to use for beginners. The following example shows the bare minimum needed to reserve URL prefixes for ports 80 and 443:
+
+```console
+netsh http add urlacl url=http://+:80/ user=Users
+netsh http add urlacl url=https://+:443/ user=Users
+```
+
+The following example shows how to assign an SSL certificate:
+
+```console
+netsh http add sslcert ipport=0.0.0.0:443 certhash=MyCertHash_Here appid={00000000-0000-0000-0000-000000000000}".
+```
+
+Here is the official reference documentation:
+
+* [Netsh Commands for Hypertext Transfer Protocol (HTTP)](http://technet.microsoft.com/library/cc725882.aspx)
+* [UrlPrefix Strings](https://msdn.microsoft.com/library/windows/desktop/aa364698.aspx)
+
+The following resources provide detailed instructions for several scenarios. Articles that refer to `HttpListener` apply equally to `HttpSys`, as both are based on Http.Sys.
+
+* [How to: Configure a Port with an SSL Certificate](http://msdn.microsoft.com/library/ms733791.aspx)
+* [HTTPS Communication - HttpListener based Hosting and Client Certification](http://sunshaking.blogspot.com/2012/11/https-communication-httplistener-based.html) This is a third-party blog and is fairly old but still has useful information.
+* [How To: Walkthrough Using HttpListener or Http Server unmanaged code (C++) as an SSL Simple Server](http://blogs.msdn.com/b/jpsanders/archive/2009/09/29/walkthrough-using-httplistener-as-an-ssl-simple-server.aspx) This too is an older blog with useful information.
+* [How Do I Set Up A .NET Core HttpSys With SSL?](https://blogs.msdn.microsoft.com/timomta/2016/11/04/how-do-i-set-up-a-net-core-HttpSys-with-ssl/)
+
+Here are some third-party tools that can be easier to use than the netsh.exe command line. These are not provided by or endorsed by Microsoft. The tools run as administrator by default, since netsh.exe itself requires administrator privileges.
+
+* [HttpSysManager](http://httpsysmanager.codeplex.com/) provides UI for listing and configuring SSL certificates and options, prefix reservations, and certificate trust lists. 
+* [HttpConfig](http://www.stevestechspot.com/ABetterHttpcfg.aspx) lets you list or configure SSL certificates and URL prefixes. The UI is more refined than HttpSysManager and exposes a few more configuration options, but otherwise it provides similar functionality. It cannot create a new certificate trust list (CTL), but can assign existing ones.
+
+For generating self-signed SSL certificates, Microsoft provides command-line tools: [MakeCert.exe](https://msdn.microsoft.com/library/windows/desktop/aa386968) and the PowerShell cmdlet [New-SelfSignedCertificate](https://technet.microsoft.com/library/hh848633). There are also third-party UI tools that make it easier for you to generate self-signed SSL certificates:
```

Comment:
I see. I have a few that are setup wrong from several months ago, including one in the Azure Key Vault topic I'm working now that I'll change. It has bothered me, so I'm looking forward to getting the styling off of any prior links I created.